pkg/server/server_test.go

```diff
@@ -225,6 +225,21 @@ func TestLokiConfiguration(t *testing.T) {
 	body, err := ioutil.ReadAll(resp.Body)
 	require.NoError(t, err)
 	assert.Equal(t, `{"hello":"world!"}`, string(body))
+
+	// WHEN the Loki flows endpoint is queried in the backend
+	resp, err = backendSvc.Client().Get(backendSvc.URL + "/api/loki/topology")
+	require.NoError(t, err)
+
+	// THEN the query has been properly forwarded to Loki
+	req = lokiMock.Calls[1].Arguments[1].(*http.Request)
+	assert.Equal(t, `topk(100,sum by(SrcK8S_Name,SrcK8S_Type,SrcK8S_OwnerName,SrcK8S_OwnerType,SrcK8S_Namespace,SrcAddr,DstK8S_Name,DstK8S_Type,DstK8S_OwnerName,DstK8S_OwnerType,DstK8S_Namespace,DstAddr) (sum_over_time({app="netobserv-flowcollector"}|json|unwrap Bytes|__error__=""[300s])))`, req.URL.Query().Get("query"))
```

Comment:
Should the space in the middle (`,DstAddr) (sum_over_time(`) have been sent as a `%20` or a `+`?

Reply:
(or maybe has been already decoded by the `Query().Get` function?)

Reply:
Everything is encoded in [EncodeQuery](https://github.com/netobserv/network-observability-console-plugin/pull/101/files#diff-f4c6e4b4bb7c8a94546df914625e18dfae6262e7a4a8841e30fdec1de0973b32R105) function

Reply:
I prefer to encode at very end to allow clear logs = copy / paste for testing